charts/app-config/templates/external-secrets/content-data-admin/google.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if not (has $.Values.govukEnvironment (list "staging" "production")) }}
```

Comment:
Can we add them for all the envs so we don't trip over this later?

Reply:
Copy and paste error - good spot - yes we can add for all and we should.

Reply:
but i would need to then create the relevant secret for all ENVS

Reply:
Yeah that's true, though it's a good idea to do all the envs now because otherwise we'll trip over them later and spend ages debugging unnecessarily (this has been such a bugbear for me the last couple of days trying to troubleshooting the publishing-api/content-store stuff).

Reply:
i will update them in a separate PR. where i will go through all the app secrets one by one. all ENVS